src/shared/modules/dbMeta/dbMetaDuck.js

```diff
@@ -104,6 +104,7 @@ export const allowOutgoingConnections = state =>
 export const credentialsTimeout = state =>
   getAvailableSettings(state)['browser.credential_timeout'] || 0
 export const getRemoteContentHostnameWhitelist = state =>
```

Comment:
Rename this too please

Reply:
@oskarhane We say whitelist in tons of places, so it felt inconsistent to only rename it in one method. Should I go ahead and change all whitelist->allowlist or just this on?

Reply:
@OskarDamkjaer Yes, please